plugins/modules/aci_l3out_hsrp_group.py

```diff
@@ -0,0 +1,393 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2023, Shreyas Srish (@shrsr) <ssrish@cisco.com>
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "community"}
+
+DOCUMENTATION = r"""
+---
+module: aci_l3out_hsrp_group
+short_description: Manage HSRP group (hsrp:GroupP) of the HSRP interface profile (hsrp:IfP)
+description:
+- Manage HSRP group of the HSRP interface profile on Cisco ACI fabrics.
+options:
+  tenant:
+    description:
+    - Name of an existing tenant.
+    type: str
+    aliases: [ tenant_name ]
+  l3out:
+    description:
+    - Name of an existing L3Out.
+    type: str
+    aliases: [ l3out_name ]
+  node_profile:
+    description:
+    - Name of the node profile.
+    type: str
+    aliases: [ node_profile_name, logical_node ]
+  interface_profile:
+    description:
+    - Name of an existing interface profile.
+    type: str
+    aliases: [ name, interface_profile_name, logical_interface ]
+  hsrp_interface_group:
+    description:
+    - Name of the HSRP interface group.
+    type: str
+    aliases: [ name, hsrp_group ]
+  group_id:
+    description:
+    - The group id of the HSRP interface group.
+    type: int
+  ip:
+    description:
+    - The virtual IP address of the HSRP interface group.
+    type: str
+  mac:
+    description:
+    - The MAC address of the HSRP interface group.
+    type: str
+  group_name:
+    description:
+    - The group name of the HSRP interface group.
```

Comment:
Can we change the description of this attribute to make it clearer? It is a bit confusing regarding what is the description of `hsrp_interface_group`.

Reply:
Ok